concertx_sol/programs/concert-x/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub mod concert_x {
 pub struct CreateConcert<'info> {
     #[account(
         init,
-        seeds = [b"concertX", initializer.key().as_ref()],
+        seeds = [b"concertX", title.as_bytes(), initializer.key().as_ref()],
```

Comment:
@jucoba wondering if the seeds could be a combination between the initializer and the title? So other creators can create concerts with the same title as well 🤔